src/concrete/ml/sklearn/base.py

```diff
@@ -2086,9 +2086,9 @@ def get_topk_labels(self, X: Data, fhe: Union[FheMode, str] = FheMode.DISABLE) -
 
         topk_labels = []
         for query in X:
-            topk_labels.append(BaseEstimator.predict(self, query[None], fhe=fhe))
+            topk_labels.append(BaseEstimator.predict(self, numpy.expand_dims(query, 0), fhe=fhe)[0])
```

Comment:
you can maybe separate this into two lines as we usually do, it would make this more readable !